tools/amd_build/patches/a_aten_src_THCUNN_generic_SpatialCrossMapLRN.cu.patch

```diff
@@ -0,0 +1,19 @@
+diff --git a/aten/src/THCUNN/generic/SpatialCrossMapLRN.cu b/aten/src/THCUNN/generic/SpatialCrossMapLRN.cu
+index 61a1c7046..222ec9b87 100644
+--- a/aten/src/THCUNN/generic/SpatialCrossMapLRN.cu
++++ b/aten/src/THCUNN/generic/SpatialCrossMapLRN.cu
+@@ -80,12 +80,14 @@ void LRNbackward(THCState* state, THCTensor* input, THCTensor* output,
+   gradOutput = THCTensor_(newContiguous)(state, gradOutput);
+ 
+   int n_threads = batchSize * imsize_h * imsize_w;
++#if defined(__NVCC__)
```

Comment:
So, HIP version of this just doesn't work, is that right? Wouldn't it be better to raise an error than just skip the kernel?